src/Domain.Schema/Converters/SensorMeasurementConverter.cs

```diff
@@ -8,16 +8,38 @@
 using OMP.Connector.Domain.Schema.Interfaces;
 using OMP.Connector.Domain.Schema.SensorTelemetry;
 using OMP.Connector.Domain.Schema.SensorTelemetry.PrimitiveTypes.Base;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace OMP.Connector.Domain.Schema.Converters
 {
     public class SensorMeasurementConverter : CustomJsonConverter<IMeasurementValue>
     {
+        public override bool CanWrite => true;
 
         protected override bool LoadAfterCreate => false;
 
         public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
-            => serializer.Serialize(writer, value, value.GetType());
+        {
+            if(value is List<SensorMeasurement> list)
+            {
+                foreach (var item in list)
+                {
+                    if(item.Key.Equals("Value"))
```

Comment:
Is there some code missing here? If not can we remove the empty if block?

Reply:
I don't think this is a change that is supposed to go into this PR.
I have removed the change.